services/mozilla-observatory/mozilla-observatory.service.tester.js

```diff
@@ -0,0 +1,224 @@
+'use strict'
+
+const { ServiceTester } = require('..')
+
+const t = (module.exports = new ServiceTester({
+  id: 'mozilla-observatory',
+  title: 'Mozilla Observatory Scanner',
+}))
```

Comment:
We can use the short hand helper function instead. Replace lines 3-8 with:

```
const t = (module.exports = require('../tester').createServiceTester())
```

Reply:
I don't understand why I get a `Error: Unknown services: mozilla-observatory` 🤔 

See 8813d49c43260d57a25d687713156b107b0672a8

Reply:
The token created by the shorthand excludes the `-` character, so to run these tests it will be `MozillaObservatory` (case insensitive). 

I've updated the PR title accordingly

Reply:
Cool thank you `npm run test:services -- --only=MozillaObservatory` is working 😉